trl/trainer/dpo_trainer.py

```diff
@@ -456,6 +456,39 @@ def dpo_loss(
 
         return losses, chosen_rewards, rejected_rewards
 
+    def ipo_loss(
+        self,
+        policy_chosen_logps: torch.FloatTensor,
+        policy_rejected_logps: torch.FloatTensor,
+        reference_chosen_logps: torch.FloatTensor,
+        reference_rejected_logps: torch.FloatTensor,
+    ) -> Tuple[torch.FloatTensor, torch.FloatTensor, torch.FloatTensor]:
+        """Compute the IPO loss for a batch of policy and reference model log probabilities.
+        The beta is the regularization parameter for the IPO loss, denoted by tau in the paper.
```

Comment:
Similar comment as a above, I would move this to the code below where we define the equation. 

We could describe what `beta` is in the docs of the trainer attributes

Reply:
added to the docstring of the Trainer too